numpy/core/tests/test_indexing.py

```diff
@@ -714,7 +709,8 @@ def setUp(self):
             0,
             # Boolean indices, up to 3-d for some special cases of eating up
             # dimensions, also need to test all False
-            np.array(False),
+            # Fixme
```

Comment:
I had a glance at the code, I think I can probably fix it with two small modifications.